### PR TITLE
add a new line to file2

### PR DESCRIPTION
**Pull request by azikram**
## check the change(s) made:
* file2, line 3

### CODE REVIEW

**Comment by azikram** [comment]
looks good?

**Comment by Shahriar-exoldius** [comment]
Naise

**Review by Shahriar-exoldius — CHANGES_REQUESTED** [review]
Write Sabbir!

File: file2.js [hunk 1]
```diff
@@ -1 +1,3 @@
 pull
+
+hello, Shahriar
```
Comment (azikram) [inline file2.js:3]: looks okay, right?

Comment (Shahriar-exoldius) [reply file2.js:3]: echo "Sabbir Dnj"

**Comment by azikram** [comment]
> Write Sabbir!

Done.

**Review by azikram — COMMENTED** [review]
changes made.

**Pull request merged**